modules/user-guide/pages/analyze-vale-app.adoc

```diff
@@ -0,0 +1,25 @@
+// Metadata for Antora
+:navtitle: Analyzing with the Red Hat Vale app
```

Comment:
```suggestion
:navtitle: Using Vale-at-Red-Hat online
```

Reply:
Can we standardize/simplify by using the title of the app, "Vale-at-Red-Hat online", throughout?